ag-grid-webcomponent/index.js

```diff
@@ -119,6 +119,15 @@ class AgGrid extends HTMLElement {
     }
   }
 
+  set selectedIds(selectedIds) {
+    if (selectedIds.length == 0) this.api.deselectAll();
+
+    this.api.forEachNode(function (node) {
```

Comment:
This could be moved into an else case. No need to execute this block when length is 0.